### PR TITLE
Close sockets after ssl teardown

### PR DESCRIPTION
**Pull request by msgmaxim**

### CODE REVIEW

File: httpserver/http_connection.cpp [hunk 1]
```diff
@@ -186,10 +188,20 @@ connection_t::connection_t(boost::asio::io_context& ioc, ssl::context& ssl_ctx,
...
     start_timestamp_ = std::chrono::steady_clock::now();
 }
 
-connection_t::~connection_t() { LOKI_LOG(trace, "~connection_t"); }
+connection_t::~connection_t() {
+
+    // Safety net
+    if (stream_.lowest_layer().is_open()) {
+        LOKI_LOG(warn, "Client socket should be closed by this point, but wasn't. Closing now.");
+        stream_.lowest_layer().close();
```
Comment (BeaudanBrown) [inline httpserver/http_connection.cpp]: Should we maybe move this `is_open` check to `do_close`, and just call `do_close` here?

File: httpserver/http_connection.cpp [hunk 2]
```diff
@@ -1104,6 +1117,7 @@ void HttpClientSession::start() {
 
 void HttpClientSession::trigger_callback(SNodeError error,
                                          std::shared_ptr<std::string>&& body) {
+    LOKI_LOG(trace, "trigger callback");
```
Comment (BeaudanBrown) [inline httpserver/http_connection.cpp]: Not important but would like to try have all new logs (and retroactively if we see them) start with a capital letter... 
# :confetti_ball:

File: httpserver/service_node.cpp [hunk 3]
```diff
@@ -531,7 +531,7 @@ parse_swarm_update(const std::shared_ptr<std::string>& response_body) {
 }
 
 void ServiceNode::swarm_timer_tick() {
-    LOKI_LOG(trace, "UPDATING SWARMS: begin");
+    LOKI_LOG(trace, "swarm timer tick");
```
Comment (BeaudanBrown) [inline httpserver/service_node.cpp]:
```suggestion
    LOKI_LOG(trace, "Swarm timer tick");
```
# :grimacing:

**Comment by sachaaaaa** [comment]
> We're a bit stabbing in the dark but I guess it can't hurt!

Imposter !

File: httpserver/http_connection.cpp (hunk 1 above)
Comment (msgmaxim) [reply httpserver/http_connection.cpp]: Will do in the upcoming PR!

**Pull request merged**